R/gta setwd.R

```diff
@@ -19,11 +19,13 @@ gta_setwd = function(gdrive=NA){
 
   } else {
 
-    gdrive.gta_cloud <- "/.shortcut-targets-by-id/17FRvsAcpS6vSAA-JfUUBOS0ixF9FSuio/GTA cloud"
```

Comment:
I think we have to keep this line of code. If we put the assignment of the variable inside the if condition, it won't work for windows case.